orator/query/builder.py

```diff
@@ -353,6 +353,15 @@ def where(self, column, operator=Null(), value=None, boolean='and'):
         if isinstance(column, QueryBuilder):
             return self.where_nested(column, boolean)
 
+        if isinstance(column, list):
+            nested = self.new_query()
+            for condition in column:
+                if isinstance(condition, list) and len(condition) == 3:
+                    nested.where(condition[0], condition[1], condition[2])
+                else:
+                    raise ArgumentError("Invalid inputs in the list")
```

Comment:
I think the error message here should more explicit.

Something like "Invalid conditions in where() clause.".